test/util/mocks/TokenValidFunctions.js

```diff
@@ -13,6 +13,7 @@ function initializeTokenMock() {
   checkifTokenValidMock = sinon.stub(TokenFunctions, 'checkIfTokenValid');
   decodeTokenValidMock = sinon.stub(TokenFunctions, 'decodeToken');
   verifyTokenMock = sinon.stub(TokenValidation, 'verifyToken');
+  discordApiKeyMock = sinon.stub(TokenValidation, 'checkDiscordKey');
```

Comment:
token and discord are different things, since only 1 api rn cares about discord api keys, move this back into the user.js test file